src/index.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
 import store from './redux/configureStore';
+// import { getBooks } from './redux/books/books';
 import './index.css';
 import App from './App';
 
+// store.dispatch(getBooks);
```

Comment:
- [ ] **Required:** You have left some commented code in your codebase. It is unprofessional to commit commented code. Kindly remove them throughout your codebase.